app/helpers/invite_tool_message_helper.rb

```diff
@@ -1,17 +1,17 @@
 module InviteToolMessageHelper
   def default_assessment_message
-    'The goal of this assessment is to discuss professional development and determine areas of strength or limitations within your district. To participate along with your colleagues, you must first register with the PD Readiness Assessment Tool. Once registered, you’ll be able to answer the questions and have on-going access to the assessment discussions.'
+    'The goal of this assessment is to discuss professional development and determine areas of strength or limitations within your district. To participate along with your colleagues, you must first register with PDredesign. Once registered, you’ll be able to answer the questions and have on-going access to the Assessment discussions.'
   end
 
   def default_inventory_message
-    "The goal of this inventory is to get the big picture of data and technology systems that support professional learning in your district. The inventory will be followed by an in-person discussion of how well current data systems, technology infrastructure, and product purchases are aligned to professional development goals.
+    "The goal of the PDredesign Data & Tech Inventory is to get the big picture of data and technology systems that support professional learning in your district.
```

Comment:
This may cause some confusion on staging; the message provided here is the default but we do not take this and update it with any existing entities.  There's no bug here; just something to keep in mind while we're testing.